source/AI.cpp

```diff
@@ -980,7 +980,7 @@ void AI::Step(const PlayerInfo &player, Command &activeCommands)
 				&& parent->Position().Distance(it->Position()) > 500.)
 			MoveEscort(*it, command);
 		// Otherwise, attack targets depending on how heroic you are.
```

Comment:
If it's not `heroic` anymore, this should be changed.
```suggestion
		// Otherwise, attack targets depending on your hunting attribute.
```